README.md

```diff
@@ -39,30 +40,24 @@ It should look like this
 ```ruby
 RSpec.configure do |config|
   # Existing code
+  config.after(:each, type: :controller) do
+    SmashingDocs.run!(request, response, true)
+  end
   config.after(:suite) { SmashingDocs.finish! }
 end
 ```
-#### To run on all controller tests
 
-Add this to your `spec_helper.rb`
+#### To run on only select tests
+Set the `c.run_all` line to `false` in `rails_helper.rb`
 ```ruby
-config.after(:each, type: :controller) do
-  SmashingDocs.run!(request, response)
+SmashingDocs.config do |c|
+  c.template_file = 'smashing_docs/template.md'
+  c.output_file   = 'smashing_docs/api_docs.md'
+  c.run_all       = false
 end
 ```
 
-The whole file should look like this
-```ruby
-RSpec.configure do |config|
-  # Existing code
-  config.after(:each, type: :controller) do
-    SmashingDocs.run!(request, response)
-  end
-  config.after(:suite) { SmashingDocs.finish! }
-end
-```
-#### To run on only select tests
-Just add `SmashingDocs.run!(request, response)` to specific tests
+Then just add `SmashingDocs.run!(request, response)` to the tests you want to run
```

Comment:
I may be able to change that in the future. I agree that it would make some tests cleaner.